packages/ui/src/Stepper/Step.tsx

```diff
@@ -15,6 +17,7 @@ interface IStepProperties {
   index: number;
   label?: string;
   lineStyle?: LineStyleType;
+  listalignment?: ListAlignmentType;
```

Comment:
Should we call it just `align`?
```suggestion
  align?: AlignType;
```